angular-client/components/tasks.js

```diff
@@ -0,0 +1,14 @@
+angular.module('meepo')
+.component('tasks', 
+{
+	controller: function()
+	{
```

Comment:
Generally, functions should be defined with the opening bracket on the same line as the 'function' keyword. Example:

```
  controller: function() {
    var num = 1;
    return num;
  },
```